proposals/android/README.md

```diff
@@ -0,0 +1,42 @@
+# Introduction
+
+As we know, the importance of mobile devices graphics is increasing for this digital world. There are various types of mobile devices that require presentation, interaction, visualization, and other related graphics  is one of major mobile device markets.
+We propose a new rendering pipeline to extend the USD capabilities to support Android platform with Vulkan backend. The new solution helps benefit Android users from the USD ecosystem.
+
+# Technical challenges:
+
+- USD does not support Android including build system, rendering pipeline, lib plugin, file loader system and so on
+- hgiVulkan backend does not support Android specific function and parameters
+- The Vulkan version in hgiVulkan is too high, resulting in fewer machines that can be adapted
+- Currently there is no Android demo app
+
+
+
+# Implementation 
+
+## Development environment
+- IDE: Android Studio Flamingo | 2022.2.1 Patch 2
+- Development Kit: NDK 25.2.9519653
+
```

Comment:
Is there anything to say about support? Like, is this version a minimum expectation, or is it expected to be the only version that's supported?

Reply:
This NDK 25.2.9519653 represents this initial Android implementation and what we've validated against. In time, we anticipate building on this work to support different versions - TBD. We welcome any additional solutions to get there sooner.